crates/nu-cli/Cargo.toml

```diff
@@ -24,9 +24,11 @@ miette = { version = "5.1.0", features = ["fancy"] }
 thiserror = "1.0.31"
 fuzzy-matcher = "0.3.7"
 
-log = "0.4"
-is_executable = "1.0.1"
 chrono = "0.4.19"
+is_executable = "1.0.1"
+lazy_static = "1.4.0"
```

Comment:
A little think about this: what about using `once_cell` crate instead of `laze_static`?

I remember `once_cell` is accepted in rust std: https://doc.rust-lang.org/stable/std/lazy/struct.OnceCell.html

So after it's landed in std stable, we can save this dependency

Reply:
Good idea. I considered `once_cell` for that reason but Nu is already using `lazy_static` so I decided to use that instead of adding a new dependency.

Reply:
I think changing all `lazy_static` uses to `once_cell` would be good for another PR.

Reply:
Oh, that's fair, sorry for that I haven't check `lazy_static` is already used in `nu`, now I have no problem about it